### PR TITLE
fix: add Pranay202

### LINKED ISSUE #1161: Pranay Kharabe
# Profile Screenshot
Pranay202

### PR DESCRIPTION
**Pull request by pranay202**
resolves #1161 

### Checklist for profile addition

- [x]  Name the PR fix: add <your-username>.
- [x] You've added file named your-username.json in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x]  You've linked the PR to the correct issue.

<hr/>

### Other additions/changes

- [x]  Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x]  I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Pranay202.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/68046838/172583862-aab04eeb-d77f-42b3-8678-b05a23daf57e.png",
+    "issueId": 1161,
+    "name": "Pranay Kharabe",
+    "username": "Pranay202"
+}
```
Comment (Panquesito7) [inline profiles/Pranay202.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Pranay202.json (hunk 1 above)
Comment (pranay202) [reply profiles/Pranay202.json]: updated profiles/Pranayk.json

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Please remove the extra, otherwise looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me, thank you for your contribution!! :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by pranay202** [comment]
Please merge the branch then :)

**Review by kabir0x23 — APPROVED** [review]
Looks good 👍

**Review by kabir0x23 — APPROVED** [review]
Looks good 👍
Nice profile 🌌

**Review by Panquesito7 — DISMISSED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: profiles/Pranay202.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/68046838/172583862-aab04eeb-d77f-42b3-8678-b05a23daf57e.png",
+    "issueId": 1161,
+    "name": "Pranay Kharabe",
+    "username": "Pranay202"
```
Comment (Panquesito7) [inline profiles/Pranay202.json:5]: Oh well, now that I noticed, it looks like there's an indentation issue here.
```suggestion
  "image": "https://user-images.githubusercontent.com/68046838/172583862-aab04eeb-d77f-42b3-8678-b05a23daf57e.png",
  "issueId": 1161,
  "name": "Pranay Kharabe",
  "username": "Pranay202"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Pranay202.json (hunk 2 above)
Comment (pranay202) [reply profiles/Pranay202.json]: I have updated it according to the suggested changes. Please review @Panquesito7

**Comment by pranay202** [comment]
> Looks good 👍 Nice profile 🌌

Thank you so much Kabir.

**Review by Panquesito7 — APPROVED** [review]
Thank you! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Maintainable

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**